src/nvim/ex_cmds2.c

```diff
@@ -551,7 +553,7 @@ void ex_debuggreedy(exarg_T *eap)
 void ex_breakdel(exarg_T *eap)
 {
   struct debuggy *bp, *bpi;
-  int nr;
+  long nr;
```

Comment:
Possible if I change `atol((char *)eap->arg)` below this to `atoi((char *)eap->arg)`. Has same effect as casting the result to int. Done.